x/gov/spec/02_state.md

```diff
@@ -16,11 +16,11 @@ unique id and contains a series of timestamps: `submit_time`, `deposit_end_time`
 
 A proposal will generally require more than just a set of messages to explain its
 purpose but need some greater justification and allow a means for interested participants
-to discuss and debate the proposal. In most cases, it is encouraged to have an off-chain
-system that supports the on-chain governance process. To accommodate for this, a
-proposal contains a special `metadata` field, an array of bytes, which can be used to
-add context to the proposal. The `metadata` field allows custom use for networks, however,
-it is expected that the field contains a URL or some form of CID using a system such as
+to discuss and debate the proposal.
+In most cases, **it is encouraged to have an off-chain system that supports the on-chain governance process**.
+To accommodate for this, a proposal contains a special **`metadata`** field, an array of bytes,
```

Comment:
I know this PR is closed but isn't this a string instead of an array of bytes @julienrbrt ?

Reply:
`metadata` is a string, yes.

Reply:
cc @julienrbrt

Reply:
Fix incoming...

Reply:
https://github.com/cosmos/cosmos-sdk/pull/12837